docs/content/_navigation.json

```diff
@@ -1135,6 +1139,10 @@
             "title": "Papertrail (dagster-papertrail)",
             "path": "/_apidocs/libraries/dagster-papertrail"
           },
+          {
+            "title": "Pipes (dagster-pipes)",
```

Comment:
same - should this be "Dagster Pipes"? so it doesn't sound like an integration lib

Reply:
I think here you're right and it should be "Dagster Pipes".

I was struggling with where to put it because I couldn't find either `dagster-graphql` or `dagster-webserver` listed, and it would've been natural to group it with them.

Reply:
that's a good point. lets just do "Dagster Pipes" for now. i agree that it should be the same group with dagster-graphql and dagster-webserver. cc @erinkcochran87 getting this on your radar in terms of api doc nav.